src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Tab.scss';
```

Comment:
I guess the name of this component is TabItem. The Tab component for me is the combinaisons of all TabItem ?